src/applications/ivc-champva/10-7959f-1/config/form.js

```diff
@@ -112,7 +112,7 @@ const formConfig = {
       title: 'Name and date of birth',
       pages: {
         page1: {
-          // initialData: mockdata.data,
+          initialData: mockdata.data,
```

Comment:
@reiting Did you mean to include this?

Reply:
WHOOPS! Hahaha I totally forgot to comment back out. Fixed now!